editor/src/components/canvas/controls/classname-select.tsx

```diff
@@ -171,11 +171,49 @@ const ValueContainer = betterReactMemo(
   },
 )
 
-const filterOption = createFilter({ ignoreAccents: false })
+const filterOption = () => true
+const MaxResults = 500
 
 export const ClassNameSelect: React.FunctionComponent = betterReactMemo('ClassNameSelect', () => {
   const theme = useColorTheme()
   const dispatch = useEditorState((store) => store.dispatch, 'ClassNameSelect dispatch')
+  const [input, setInput] = React.useState('')
+  const filteredOptions = React.useMemo(() => {
+    const trimmedLowerCaseInput = input.trim().toLowerCase()
+    if (trimmedLowerCaseInput === '') {
+      return TailWindOptions.slice(0, MaxResults)
+    } else {
+      // First find all matches, and use a sparse array to keep the best matches at the front
+      let orderedMatchedResults: Array<Array<TailWindOption>> = []
+      let perfectMatchCount = 0
+      for (var i = 0; i < TailWindOptions.length && perfectMatchCount < MaxResults; i++) {
```

Comment:
Should this be checking the length of `existingMatched` against `MaxResults`?

Reply:
Nah, this was deliberate, so that we don't cut the search too early. In this case we only want to end the search early if we've already found the maximum number of perfect matches